n2khabutils/R/textdata.R

```diff
@@ -374,6 +378,13 @@ NULL
 #'     }
 #'   }
 #'
+#' \itemize{
+#'     \item{For the MHQ-schemes, the list of sampled types is based on the report of
+#'     \href{https://pureportal.inbo.be/portal/files/4339795/Westra_etal_2014_MonitoringNatura2000Habitats.pdf}{Westra \emph{et al.} (2014)}.
```

Comment:
You may want to add the same source at line 293 as well (documentation of `schemes`).